notebooks/Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -787,76 +897,599 @@
+    "\n",
+    "**bkg_mod** - The background model returned by the function parse_bck_csv()\n",
+    "\n",
+    "**ps_mods** - List of corresponding point source models for each of the astronomical objects listed in **PSnames**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bkg_df, bkg_name, PSnames, bkg_mod, ps_mods = parse_bkg_csv(bkg_fname, solid_ang_dpi,\\\n",
+    "                    ebins0, ebins1, bl_dmask, rt_dir)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The background model **bkg_mod** is added to a list of background models **bkg_mod_list**. If there are any point source models in **ps_mods** they are added to **bkg_mod_list** and has_deriv is set to False for each of them. With all the models compiled into a single list, the background model is redefined as a compound model combining all of the point source models with the original **bkg_mod**.\n",
+    "\n",
+    "Later on, when a minimizer object is created, it will ask models for the Jacobian of the log likelihood function with respect to the model parameters. Since the parameters are fixed, there are no such derivatives to retrieve so .has_deriv is set to False."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bkg_mod.has_deriv = False\n",
+    "bkg_mod_list = [bkg_mod]\n",
+    "Nsrcs = len(ps_mods)\n",
+    "if Nsrcs > 0:\n",
+    "    bkg_mod_list += ps_mods\n",
+    "    for ps_mod in ps_mods:\n",
+    "        ps_mod.has_deriv = False\n",
+    "    bkg_mod = CompoundModel(bkg_mod_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The trigger time is used again to retrieve a dictionary of parameters at the trigger time index. **bkg_row** refers to this dictionary of parameters. The values of the parameters of the background model are retrieved from **bkg_row** and stored as another dictionary, **bkg_params**. Below is a description of the output of calling **bkg_params**.\n",
+    "\n",
+    "The first set of parameters have the form '_bkg_rate_0', '_bkg_rate_1', etc. These rates refer to contributions to the background model from diffuse sources measured in counts per second. The number index in the parameter name refers to the corresponding energy bin for which the rate is measured (e.g the parameter '_bkg_rate_0' is the counts/s in energy bin 0 attributable to diffuse sources). \n",
+    "\n",
+    "Next are the parameters of the form '_flat_0', '_flat_1', etc. These parameters refer to the rate per detector parameters in the diffuse model, $r^{b}_{j}$. Their values are the fraction of the above background rate that is the result of the rate, $r^{b}_{j}$. (e.g '_flat_0' is the fraction of '_bkg_rate_0' attributable to the rate per detector $r^{b}_{0}$)\n",
```

Comment:
Theres still a problem with your math rendering here.

Reply:
Can you pinpoint what is being incorrectly rendered? In this block only 3 things are written out in latex: 
$r^{b}_{j}$ 
$r^{b}_{j}$
$r^{b}_{0}$

The rest is in plain text. The text in the single quotes are referring to the ends of the parameter names that are part of the output, maybe that is the confusion?

Reply:
![image](https://user-images.githubusercontent.com/20259411/202091634-934e84d6-8d9f-454b-a118-6eae74979ca5.png)

Reply:
It renders fine for me in my editor. I've rewritten that section and will commit the new edit, hopefully it fixes that rendering issue.